manifests/backend.pp

```diff
@@ -40,6 +40,10 @@
 #   This parameter is mutually exclusive with ssl_key_source
 # @param include_default_resources
 #   Sets if default sensu resources should be included
+# @param manage_agent_user
+#   Sets if the Sensu agent user should be managed
+# @param agent_user_disabled
+#   Sets if the Sensu agent user should be disabled
```

Comment:
Add something about not being applicable if `manage_agent_user` is `false`.

Reply:
COmment updated and added link to Sensu Go docs for agent TLS which is when it would be most useful to disable this unnecessary user.